src/layouts/Sites.tsx

```diff
@@ -29,8 +30,57 @@ import { EmptySitesImage, IsomerLogoNoText } from "assets"
 import { AnnouncementModal } from "features/AnnouncementModal/AnnouncementModal"
 import { SiteData } from "types/sites"
 
+const SitePreviewImage = ({ imageUrl }: { imageUrl?: string }) => {
+  if (!imageUrl) {
+    return <IsomerLogoNoText height="50%" />
+  }
+  return (
+    <Image
+      src={imageUrl}
+      height="50%"
+      fallback={<IsomerLogoNoText height="50%" />}
+    />
```

Comment:
Can we omit the if condition since the `fallback` property is already defined in the `Image` tag? 
If `imageUrl` is not provided, `IsomerLogoNoText` will be displayed as a fallback.

Reply:
There are 2 possibilities for failure. The condition checks the backend is able to retrieve a favicon from going to prod homepage and return undefined for imageUrl if its not possible. I think we should not feed a blank url to image src given that it makes a query. The fallback is when there is a imageUrl but does not lead to the image such as if the image is missing. In that case there's no way we would have known and have to rely on the fallback.

Reply:
@caando so if `imageUrl` is undefined, the `Image` component will still perform a network query?

Reply:
I think if undefined goes straight to fallback bah. can test this [here](https://chakra-ui.com/docs/components/image/usage) when we put the code 
```
<Box boxSize='sm'>
  <Image src={undefined} alt='Dan Abramov' fallback={<Text>hihi</Text>} />
</Box>
```
it renders the text properly

Reply:
Thanks @kishore03109 . I think the Image component is smart enough to avoid a network query if the imageUrl is undefined. If so, @caando  I'd suggest to omit the if condition as it doesn't add any value.

Reply:
<img width="514" alt="image" src="https://github.com/isomerpages/isomercms-frontend/assets/39144313/5aa35c55-a521-47a2-8591-c1fea5c23177">
@QiluXie @kishore03109 looks like its not smart enough and will make a request.